src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/repository/CalculationReasonRepository.kt

```diff
@@ -5,4 +5,7 @@ import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.entity.CalculationR
 
 interface CalculationReasonRepository : JpaRepository<CalculationReason, Long> {
   fun findAllByIsActiveTrueOrderById(): List<CalculationReason>
+
+  fun findTopByIsBulkTrue(): CalculationReason
```

Comment:
We should probably return an `Optional<CalculationReason>` in case something has gone wrong...